modules/storage-azure-create-storage-class.adoc

```diff
@@ -29,8 +29,14 @@ persistent volumes.
 .. Select `kubernetes.io/azure-disk` from the drop down list.
 
 ... Enter the storage account type. This corresponds to your Azure
-storage account SKU tier. Valid options are `Premium_LRS`, `Standard_LRS`,
+storage account SKU tier. Valid options are `Premium_LRS`, `PremiumV2_LRS`, `Standard_LRS`,
 `StandardSSD_LRS`, and `UltraSSD_LRS`.
++
+[IMPORTANT]
+====
+The skuname `PremiumV2_LRS` is not supported in all regions, and even in some supported regions, not all the availability zones are supported.
```

Comment:
Please rewrite as follows:

The skuname `PremiumV2_LRS` is not supported in all regions, and in some supported regions, not all of the availability zones are supported. For more information, see link:https://learn.microsoft.com/en-us/azure/virtual-machines/disks-deploy-premium-v2[Azure doc].